Makefile

```diff
@@ -41,90 +41,113 @@ endif
 
 # Main targets
 
+_fix-perms: SHELL := /bin/sh
+_fix-perms: ##@prepare Fix permissions so that directory can be cleaned
+	$(shell test -d node_modules && chmod -R 744 node_modules)
+	$(shell test -d node_modules.tmp && chmod -R 744 node_modules.tmp)
```

Comment:
because otherwise I get

```
test -d node_modules && chmod -R 744 node_modules
test -d node_modules.tmp && chmod -R 744 node_modules.tmp
Makefile:46: recipe for target '_fix-perms' failed
```